plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -242,18 +216,10 @@ export default class NodesTable extends React.Component<
           />
         }
         cellRenderer={ipRenderer}
-        minWidth={
-          TableUtil.isColWidthCustom(columnWidthsStorageKey, "host")
-            ? undefined
-            : 120
-        }
+        minWidth={hasCustomWidth("host") ? undefined : 120}
```

Comment:
I am having trouble understanding how this works. Why is minWidth undefined?

Reply:
please note that i did not change semanticts here but only aliased    `TableUtil.isColWidthCustom(columnWidthsStorageKey, column)`.

i think ui-kit wants minWidth to not be present when the user chose a custom width which could be smaller than minWidth. you could try to ask Mike or Philipp about this.